omnij-rpc/src/main/groovy/foundation/omni/consensus/OmniCoreConsensusTool.groovy

```diff
@@ -58,25 +58,10 @@ class OmniCoreConsensusTool extends ConsensusTool {
         tool.run(args.toList())
     }
 
-    private SortedMap<Address, ConsensusEntry> getConsensusForCurrency(CurrencyID currencyID) {
-        List<MPBalanceEntry> balances = client.omniGetAllBalancesForId(currencyID)
-
-        TreeMap<Address, ConsensusEntry> map = [:]
-
-        balances.each { MPBalanceEntry item ->
-
-            Address address = item.address
-            ConsensusEntry entry = itemToEntry(item)
-
-            if (address != "" && entry.balance > 0) {
```

Comment:
Ha! Omni Core doesn't return emtpy balances, but OmniChest and OmniWallet do.

The issue is now clearer, especially given the test report. The extra filtering here is needed, but it should be:
```diff
diff --git a/omnij-rpc/src/main/groovy/foundation/omni/consensus/ChestConsensusTool.groovy b/omnij-rpc/src/main/groovy/foundation/omni/consensus/ChestConsensusTool.groovy
index 4e10fb2..5fd23fe 100644
--- a/omnij-rpc/src/main/groovy/foundation/omni/consensus/ChestConsensusTool.groovy
+++ b/omnij-rpc/src/main/groovy/foundation/omni/consensus/ChestConsensusTool.groovy
@@ -51,3 +51,3 @@ class ChestConsensusTool extends ConsensusTool {
 
-            if (address != "" && entry.balance > 0) {
+            if (address != "" && (entry.balance > 0 || entry.reserved > 0)) {
                 map.put(address, entry)
diff --git a/omnij-rpc/src/main/groovy/foundation/omni/consensus/OmniwalletConsensusTool.groovy b/omnij-rpc/src/main/groovy/foundation/omni/consensus/OmniwalletConsensusTool.groovy
index 96bc46e..c9f656e 100644
--- a/omnij-rpc/src/main/groovy/foundation/omni/consensus/OmniwalletConsensusTool.groovy
+++ b/omnij-rpc/src/main/groovy/foundation/omni/consensus/OmniwalletConsensusTool.groovy
@@ -48,3 +48,3 @@ class OmniwalletConsensusTool extends ConsensusTool {
 
-            if (address != "" && entry.balance > 0) {
+            if (address != "" && (entry.balance > 0 || entry.reserved > 0)) {
                 map.put(address, entry)
```

Up until now, we basically trimmed balance entries, which only have a reserved balance.